<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="shadow_frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(81, 47, 154, 255));
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 16pt;
font-family: &quot;Segoe UI Light&quot;;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Einstellungen</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Ändern der Standardprogrammeinstellungen</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollArea {
background-color: rgba(0, 0, 0, 0);
}

QLineEdit {
height: 20px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
padding-left: 10px;
padding-bottom: 5px;
border-bottom: 2px solid rgb(255, 255, 255);
border-radius: 0px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QLineEdit:hover {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QLineEdit:focus {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QLabel {
background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 8pt;
font-family: &quot;Segoe UI Light&quot;;
}

QScrollBar:vertical {
background-color: rgba(0, 0, 0, 0);
width: 15px;
margin: 15px 3px 15px 3px;
border: 1px transparent #2A2929;
border-radius: 4px;
}

QScrollBar::handle:vertical {
background-color: #fff;
min-height: 5px;
border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
background-color: rgba(0, 255, 213);
}

QScrollBar::sub-line:vertical {
margin: 3px 0px 3px 0px;
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
margin: 3px 0px 3px 0px;
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on {
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on {
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
}</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-488</y>
            <width>375</width>
            <height>938</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Gruppenname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Voraussetzung ist das TDMS-Dateiformat. Um den Gruppennamen zu erhalten, öffnen Sie die Datei mit dem Excel Importer.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_group_name">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Gruppenname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Kanalname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Voraussetzung ist das TDMS-Dateiformat.Um den Kanalnamen zu erhalten, öffnen Sie die Datei mit dem Excel Importer.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_channel_name">
             <property name="font">
              <font>
               <family>Segoe UI Light</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Kanalname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Abtastrate [Hz]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Im Experiment verwendete Abtastrate [Hz].</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_sampling_rate">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Abtastrate [Hz]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Resampling Faktor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Aufgrund der Aufzeichnungslängen ist die Berechnungszeit relativ aufwändig, daher ist ein Downsampling erforderlich.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_resample_factor">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Resampling Faktor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Fenstergröße</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Anzahl der Werte über die gemittelt werden soll (siehe Median, arithmetischer und quadratischer Mittelwert).</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_window_size">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Fenstergröße</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Leerlaufbeginn [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Standardeinstellung für den ersten Messwertdes Leerlaufs.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_idle_0">
             <property name="placeholderText">
              <string>Leerlaufbeginn [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Leerlaufende [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Standardeinstellung für den letzten Messwert des Leerlaufs.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_idle_1">
             <property name="placeholderText">
              <string>Leerlaufende [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Schnittprozessbeginn [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Standardeinstellung für den ersten Messwert des Schnittprozesses.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_cutting_0">
             <property name="placeholderText">
              <string>Schnittprozessbeginn [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Schnittprozessende [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Standardeinstellung für den letzten Messwert des Schnittprozesses.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_cutting_1">
             <property name="placeholderText">
              <string>Schnittprozessende [S]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Werkzeuge CSV-Dateiname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Name der externen CSV-Datei, die Parameter der Werkzeuge enthält.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_leitz_tools">
             <property name="placeholderText">
              <string>Werkzeuge CSV-Dateiname</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color: rgb(255, 170, 0);
height: 30px;
padding-bottom: 5px;
border-radius: 17px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255)
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(255, 170, 0, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Schließen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAccept">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color:  rgb(0, 221, 51);
height: 30px;
padding-bottom: 5px;
border-radius: 17px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255)
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(0, 221, 51, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Bestätigen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
